script/foreman-debug

```diff
@@ -303,6 +329,28 @@ add_files /etc/sysconfig/pgsql
 add_files "/var/lib/pgsql/data/pg_log/*"
 add_cmd "foreman-rake plugin:list" "plugin_list"
 
+if [ $DETAILED -eq 1 ]; then
+  printv "Adding detailed information"
+  add_sql "select id,name,value from settings where name not similar to '%(pass|key|secret)' order by name" settings
+  add_sql "select type,name,host,port,account,base_dn,attr_login,onthefly_register,tls from auth_sources order by name" auth_sources
+  add_sql "select id,name,type from compute_resources order by name" compute_resources
+  add_sql "select id,name,fullname,dns_id from domains order by name" domains
+  add_sql "select id,name,hosts_count from environments order by name" environments
+  add_sql "select id,name as name,managed,build,organization_id,location_id from hosts order by name" hosts
+  add_sql "select id,name,vm_defaults from hostgroups order by name" hostgroups
+  add_sql "select id,name,path,media_path,config_path,image_path,os_family from media order by name" media
+  add_sql "select id,host_id,mac,ip,type,name,subnet_id,domain_id from nics order by host_id,id" nics
+  add_sql "select id,name,major,minor,type,description from operatingsystems order by name" operatingsystems
+  add_sql "select id,name,url from smart_proxies order by name" smart_proxies
+  add_sql "select id,name,network,mask,dhcp_id,tftp_id,dns_id from subnets order by name" subnets
+  add_sql "select id,name,type from taxonomies order by name" taxonomies
+  add_sql "select id,name from provisioning_templates order by name" provisioning_templates
+  add_sql "select id,value as value,expires,host_id from tokens order by expires" tokens
+  add_sql "select id,login as login,admin,auth_source_id from users order by id" users
+  add_sql "select id,name from roles order by name" roles
+  add_sql "copy provisioning_templates(id,name,template) TO '$DIR/db_provisioning_templates.csv' (format csv)" provisioning_templates_export
```

Comment:
For the record, the last command will fail on MySQL adapter, but it wont error out, the output will only contain an error message instead of the data. All the others should work. @ohadlevy is currently helping me to test this on MySQL instance.